lbr_demos/lbr_fri_ros2_advanced_python_demos/lbr_fri_ros2_advanced_python_demos/admittance_control_node.py

```diff
@@ -22,20 +22,22 @@ def __init__(
         dx_gain: np.ndarray = np.array([1.0, 1.0, 1.0, 20.0, 40.0, 60.0]),
         smooth: float = 0.02,
     ) -> None:
-        self.chain_ = kinpy.build_serial_chain_from_urdf(
-            data=urdf_string, end_link_name=end_link_name, root_link_name=root_link_name
-        )
+        robot = optas.RobotModel(urdf_string=urdf_string)
+        J = robot.get_global_link_geometric_jacobian_function(end_link_name)
+        self.jacobian = lambda q: J(
+            q
+        ).toarray()  # toarray: convert casadi.DM -> numpy.ndarray
```

Comment:
OpTaS has new functionality, a `numpy_output` flag, see [here](https://github.com/cmower/optas/blob/963b8aa3e50d169ff9f3fae5d4c3e9040aa0adc9/optas/models.py#L1276). So here you can specify 
```python
self.jacobian = robot.get_global_link_geometric_jacobian_function(end_link_name, numpy_output=True)
```

`numpy_output` just specifies that the output of the function should always be a numpy array. This just means you can't pass in symbolic arrays (but you don't do that anyway here).

Reply:
replaced